### ISSUE TITLE
Small usability improvements

### ISSUE THREAD

**Issue by duud** [Andersbakken/rtags#429]
Emacs splits new windows on widescreens horizontally by default. This is a sensible default, it makes a better usage of screenspace on widescreens. When using functions like `rtags-find-references` with horizontal splitting enabled, the result looks like this:

```
/home/dima/tmp/linux-gc-test/kernel/sched/bfs.c:1020:8:			if (!smt_sh
ould_schedule(p, best_cpu))
/home/dima/tmp/linux-gc-test/kernel/sched/bfs.c:1472:8:			if (!smt_sh
ould_schedule(p, target_cpu))
/home/dima/tmp/linux-gc-test/kernel/sched/bfs.c:1503:7:		if (!smt_should
_schedule(p, target_cpu))
/home/dima/tmp/linux-gc-test/kernel/sched/bfs.c:3391:9:				if(!smt
_should_schedule(p, cpu))
/home/dima/tmp/linux-gc-test/kernel/sched/bfs.c:3529:8:			if (!smt_sh
ould_schedule(p, cpu)) {
/home/dima/tmp/linux-gc-test/kernel/sched/bfs.c:981:13:	static bool smt_sho
uld_schedule(struct task_struct *p, int cpu)
```

This is somewhat hard to parse. We could do two things to improve the situation.

1. Since rtags has the notion of the current project, we could strip the project directory prefix from paths in this views.
2. Make an option to force vertical splitting for rtags-mode buffers.

**Comment by Andersbakken** [comment]
I think option 1 is a really good idea. I've added it now. For option 2 I would actually recommend using this excellent mode: http://emacswiki.org/emacs/display-buffer-for-wide-screen.el

**Comment by duud** [comment]
Thx, for adding option 1.

Concerning the splitting of windows...
Emacs has already the functionality provided by
display-buffer-for-wide-screen.el build in. It's controlled by the
variable split-width-threshold. It's set to 160 by default. My
widow-with is 168 so emacs splits new windows horizontally. This default
is fine, 2 horizontally split windows with 80+ columns are much
better in terms of screen usage than 2 vertically split windows, so I
don't want to change this globally.
Since rtags-mode buffers contain long lines most of the time it's much
better to split them vertically by default, to avoid the
wrapping. Currently I'm wrapping all calls to functions like
rtags-find-references-at-point to force vertical splitting. It seems
like a useful default for me.

**Comment by bbannier** [comment]
What `rc` flags are needed to get back the old behavior (i.e. display of full paths)? This change appears to have broken [vim-rtags](https://github.com/lyuts/vim-rtags).

**Comment by Andersbakken** [comment]
Oops.

 --absolute-path or -K should do it.

Sorry about that.

Anders

On Wed, Sep 16, 2015 at 12:42 PM, Benjamin Bannier <notifications@github.com
